lib/board.rb

```diff
@@ -0,0 +1,32 @@
+require "pry"
+
+class Board
+  attr_accessor :board
+
+  def initialize
+    @board = top_row
+    build_empty_board
+  end
+
+  def stringify_board
+    <<~SQUIGGLY_HEREDOC
+      #{board}
+    SQUIGGLY_HEREDOC
```

Comment:
I think this method is adding a new line at the end of my board. Any clues why that might be? I investigated this, but didn't want to spend too much more time on it.

Reply:
There heredoc includes everything between the delimiters, including the newline after `#{board}`. Since the template isn't adding other characters, you could return `board` here without using a heredoc.

Reply:
true, good call.